test/adjoint/test_fp_iteration.py

```diff
@@ -98,7 +98,16 @@ def test_noconvergence(self):
     def test_no_late_convergence(self):
         self.parameters.drop_out_converged = True
         mesh_seq = self.mesh_seq(time_partition=TimePartition(1.0, 2, [0.5, 0.5], []))
-        mesh_seq.fixed_point_iteration(oscillating_adaptor0, parameters=self.parameters)
+        solver_kwargs = (
+            {"track_coefficients": False}
+            if type(mesh_seq) is GoalOrientedMeshSeq
+            else {}
+        )
```

Comment:
Can this not just be the following?
```python
solver_kwargs = {"track_coefficients": False}
```
Since this is the default for `AdjointMeshSeq` anyway?

Reply:
Removed this entirely in https://github.com/mesh-adaptation/goalie/pull/259/commits/1ce741b79639c1b34bf104202e5693586fc168ae